test/document_stores/test_document_store.py

```diff
@@ -1082,9 +1082,7 @@ def test_multilabel_meta_aggregations(document_store: BaseDocumentStore):
             assert multi_label.filters == l.filters
 
 
-@pytest.mark.parametrize(
-    "document_store", ["elasticsearch", "faiss", "milvus", "weaviate", "pinecone", "memory"], indirect=True
-)
+@pytest.mark.parametrize("document_store", ["memory"], indirect=True)
```

Comment:
Seems unrelated, should this change go into a InMemoryDocumentStore related PR?

Reply:
That's actually something I should've done at each refactoring for es, faiss, weaviate, pinecone and milvus of course. Not sure it's worth it a dedicated PR?